frontend/src/components/OpeningScreenDataTable/testData.js

```diff
@@ -0,0 +1,263 @@
+      openingID: '114200',
+      fileID: 'TFL47',
+      cuttingPermit: '12T',
+      timberMark: '47/12S',
+      cutBlock: '12-44A',
+      grossAreaHa: '12.9',
+      status: 'Active',
+      category: 'FTML',
+      disturbanceStart: '-',
+      createdAt: '2021-10-20',
+      lastViewed: '2022-10-24'
+    },
+    {
+      id: '114199',
+      openingID: '114199',
+      fileID: 'TFL47',
+      cuttingPermit: '12T',
+      timberMark: '47/12S',
+      cutBlock: '12-44A',
+      grossAreaHa: '12.9',
+      status: 'Active',
+      category: 'FTML',
+      disturbanceStart: '-',
+      createdAt: '2021-10-20',
+      lastViewed: '2022-10-24'
+    },
+    {
+      id: '114198',
+      openingID: '114198',
+      fileID: 'TFL47',
+      cuttingPermit: '12T',
+      timberMark: '47/12S',
+      cutBlock: '12-44A',
+      grossAreaHa: '12.9',
+      status: 'Free growing',
+      category: 'FTML',
+      disturbanceStart: '-',
+      createdAt: '2021-09-12',
+      lastViewed: '2022-10-23'
+    },
+    {
+      id: '114197',
+      openingID: '114197',
+      fileID: 'TFL47',
+      cuttingPermit: '12T',
+      timberMark: '47/12S',
+      cutBlock: '12-44A',
+      grossAreaHa: '12.9',
+      status: 'Free growing',
+      category: 'FTML',
+      disturbanceStart: '-',
+      createdAt: '2021-09-12',
+      lastViewed: '2022-10-23'
+    },
+    {
+      id: '114196',
+      openingID: '114196',
+      fileID: 'TFL47',
+      cuttingPermit: '12T',
+      timberMark: '47/12S',
+      cutBlock: '12-44A',
+      grossAreaHa: '12.9',
+      status: 'Free growing',
+      category: 'FTML',
+      disturbanceStart: '-',
+      createdAt: '2021-08-05',
+      lastViewed: '2022-10-22'
+    },
+    {
+      id: '114195',
+      openingID: '114195',
+      fileID: 'TFL47',
+      cuttingPermit: '12T',
+      timberMark: '47/12S',
+      cutBlock: '12-44A',
+      grossAreaHa: '12.9',
+      status: 'Free growing',
+      category: 'FTML',
+      disturbanceStart: '-',
+      createdAt: '2021-08-05',
+      lastViewed: '2022-10-22'
+    },
+    {
+      id: '114194',
+      openingID: '114194',
+      fileID: 'TFL47',
+      cuttingPermit: '12T',
+      timberMark: '47/12S',
+      cutBlock: '12-44A',
+      grossAreaHa: '12.9',
+      status: 'Active',
+      category: 'FTML',
+      disturbanceStart: '-',
+      createdAt: '2021-07-10',
+      lastViewed: '2022-10-21'
+    },
+    {
+      id: '114193',
+      openingID: '114193',
+      fileID: 'TFL47',
+      cuttingPermit: '12T',
+      timberMark: '47/12S',
+      cutBlock: '12-44A',
+      grossAreaHa: '12.9',
+      status: 'Active',
+      category: 'FTML',
+      disturbanceStart: '-',
+      createdAt: '2021-07-10',
+      lastViewed: '2022-10-21'
+    },
+  ];
+  
+  export const headers = [
+    {
+      key: 'openingID',
+      header: 'Opening ID',
+    },
+    {
+      key: 'fileID',
+      header: 'File ID',
+    },
+    {
+      key: 'cuttingPermit',
+      header: 'Cutting Permit',
```

Comment:
issue: These headers are not matching with Figma. The second word starting with lowercase.

Reply:
Hi @RMCampos nice catch! I have fixed these now : ) Cheers